tests/e2e/e2e_exec_test.go

```diff
@@ -11,10 +11,92 @@ import (
 	"cosmossdk.io/math"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	"github.com/ory/dockertest/v3/docker"
 )
 
-func (s *IntegrationTestSuite) execBankSend(c *chain, valIdx int, from, to, amt, fees string, expectErr bool) {
+const (
+	flagSpendLimit      = "spend-limit"
```

Comment:
Some flags I added here because comming from different places and modules from the SDK, like the `spend-limit`. I can import from the modules, but we will add a bunch of `x/<module>/client/CLI` imports. WDYT?